spec/controllers/activity_controller_spec.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe ActivityController, type: :controller do
+end
```

Comment:
Coveralls wants test coverage for the ActivityController:
https://coveralls.io/builds/36275346/source?filename=app%2Fcontrollers%2Factivity_controller.rb#L5

You need to test the #index and #show methods